docs/Component documentation/spacing.md

```diff
@@ -22,7 +22,8 @@ $spacers: (
   11: 80px,  // 5rem
   12: 96px,  // 6rem
   13: 112px, // 7rem
-  14: 160px  // 10rem
+  14: 160px, // 10rem
```

Comment:
the changes made in this files does not seem to affect the tab tile block? if not it should not be done in this commit in my oppinion :)

Reply:
It is a detail that Martin wanted to be included for the release as I would have spent 3 times more if I had to create a new branch for this tiny thingy.

Sometimes it is ok to do so when it is so minor. Otherwise I agree. If there major changes which are not connected whatsoever then they should be done in different branches.

Reply:
I'm not saying that the creation of a completely new branch would be the right solution, just extract it out in a separate commit,